packages/core/package.json

```diff
@@ -2,20 +2,17 @@
   "name": "xstate",
   "version": "4.7.8",
   "description": "Finite State Machines and Statecharts for the Modern Web.",
-  "main": "lib/index.js",
-  "module": "es/index.js",
-  "types": "lib/index.d.ts",
```

Comment:
you might be wondering why `"types"` have been removed here and in other packages - typings are distributed in `dist` directory and main entry point receives a sibling `.d.ts` file which allows TS to resolve distributed types, making `"types"` field obsolete